eng/common/templates/steps/source-build.yml

```diff
@@ -36,7 +36,7 @@ steps:
 
     ${{ coalesce(parameters.platform.buildScript, './build.sh') }} --ci \
       --configuration $buildConfig \
-      --restore --build --pack --publish \
+      --restore --build --pack --publish -bl\
```

Comment:
This needs to get pushed into Arcade - https://github.com/dotnet/source-build/issues/2090

Reply:
https://github.com/dotnet/arcade/pull/7104

Reply:
The arcade change is in place but has not been picked up in this repo yet.  Waiting for https://github.com/dotnet/fsharp/pull/11306 to be merged.

Reply:
The latest version of Arcade was merged in, this dependency is now unblocked.